src/styles/styles.scss

```diff
@@ -0,0 +1,16 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;400;600&display=swap');
+@import 'src/styles/variables';
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: $body-font;
+  font-size: $default-font-size;
+  background-color: $neutral-very-light-gray;
+  max-width: 100vw;
```

Comment:
Is this necessary? Block elements already fit the whole screen, try handle unnecesary scrolling without this.